### ISSUE TITLE
Zooming in webpage also zooms scrollbar

### ISSUE THREAD

**Issue by elijahtlang** [endeavoursc/firefox-overlay-scrollbars-win10#4]
I noticed that when I zoom in on a webpage, the scrollbar expands proportionally. Not a big deal if the user is only zooming 110-125%, but if they need to zoom 250%, for example, it gets in the way. Is there a fix for this?

**Comment by endeavoursc** [comment]
I don't think so, because with this mod the scrollbars behave like they are part of the content, and not part of the interface.

**Comment by e-t-l** [comment]
That makes sense. Would it be possible to detect the browser's level of zoom and shrink the scrollbars accordingly, such that they remain functionally the same size when the webpage is zoomed in? (Unfortunately I'm nowhere near a strong enough programmer myself to know how to do this, or if it's even possible).
